packages/ui/src/i18n/dictionaries/authenticator/th.ts

```diff
@@ -98,7 +98,7 @@ export const thDict: AuthenticatorDictionary = {
   'Sign Up with Line': 'ลงทะเบียนด้วย Line',
   'Forgot Password': 'ลืมรหัสผ่าน',
   'Have an account? ': 'มีบัญชีอยู่แล้ว? ',
-  'Incorrect username or password': 'ชื่อผู้ใช้หรือรหัสผ่านไม่ถูกต้อง',
```

Comment:
Same feedback as [here](https://github.com/aws-amplify/amplify-ui/pull/6482#discussion_r2070556016)

```suggestion
  'Incorrect username or password': 'ชื่อผู้ใช้หรือรหัสผ่านไม่ถูกต้อง',
  'Incorrect username or password.': 'ชื่อผู้ใช้หรือรหัสผ่านไม่ถูกต้อง',
```

Reply:
Thanks. I fixed it.